exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -85,6 +84,22 @@
   private static final int MAXIMUM_RECORD_COUNT_FOR_CHECK = 10000;
   private static final int BLOCKSIZE_MULTIPLE = 64 * 1024;
 
+  /**
+   * Name of nested group for Parquet's {@code MAP} type.
+   * @see <a href="https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#maps">MAP logical type</a>
+   */
+  private static final String GROUP_KEY_VALUE_NAME = "key_value";
+  /**
+   * Name of nested group for Parquet's {@code LIST} type.
+   * @see <a href="https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists">LIST logical type</a>
+   */
+  private static final String GROUP_LIST_NAME = "list";
```

Comment:
Recently Igor was adding similar constants in https://github.com/apache/drill/pull/1844, please revisit.